mono/mini/interp/transform.c

```diff
@@ -6579,7 +6620,8 @@ interp_cprop (TransformData *td)
 		last_il_offset = ins->il_offset;
 	}
 
-	interp_local_deadce (td, local_ref_count);
+	if (interp_local_deadce (td, local_ref_count))
+		goto retry;
```

Comment:
I think it would be good to put an upper bound here (can be done later). Right now it's easy to reason that `cprop` won't happen more than twice, but presumably it won't be that easy in the future. I would like to avoid bugs where transform loops forever.